Resources/Private/Templates/Flickr/TagStream.html

```diff
@@ -0,0 +1,13 @@
+<div class="flickr-stream">
+	<h3>Photo Stream for tag(s) "{tags}"</h3>
+	<ul class="clearfix">
+		<f:for each="{feed.items}" as="photo">
+			<li class="pull-left">
+				<f:render partial="Flickr/Photo" arguments="{photo: photo}" />
+				<p>
+					<f:link.action action="userStream" arguments="{userId: photo.author_id}">Show more photos by "{photo.author}"</f:link.action>
```

Comment:
that link cannot be resolved:

> Could not resolve a route and its corresponding URI for the given parameters. This may be due to referring to a not existing package / controller / action while building a link or URI. Refer to log and check the backtrace for more details

arguments:

```json
{
  "node": "live__eyJsYW5ndWFnZSI6ImVuX1VTIn0=__336d96a5-b6c6-7d81-685f-89d32c37756e",
  "@package": "Neos.Neos",
  "@controller": "Frontend\\Node",
  "@action": "show",
  "@format": "html",
  "--neos_demo-content_flickr": {
    "@package": "neos.demo",
    "@controller": "flickr",
    "@action": "userstream",
    "@format": "html",
    "userId": "187194180@N06"
  }
}
```

Reply:
Is "node" in this a correct representation for a frontend node in a route?

Reply:
ah checked out the actual 7.3 demo and it seems the way this plugin was engineered it somehow realied on plugin views .. thats probably why it wont work?

The "master" page
<img width="312" alt="image" src="https://github.com/neos/Neos.Demo/assets/85400359/fc478488-7b73-46ab-b676-8425c104e408">

structure
<img width="242" alt="image" src="https://github.com/neos/Neos.Demo/assets/85400359/62cc423b-cc65-4514-9f15-e0163411a681">

"plugin" page

<img width="312" alt="image" src="https://github.com/neos/Neos.Demo/assets/85400359/53833f3b-07eb-4019-bd87-15fd79d93738">

Reply:
i dont understand ... not even linking to the same page works:

```php
public function tagStreamAction()
{
    return $this->uriBuilder->uriFor('tagStream');
```

Reply:
Why was it a string in the first place? I fear because of this weird thing? https://github.com/neos/neos-development-collection/blob/5f80a56210ba903e484e79a99d82b457a226198f/Neos.Neos/Classes/Routing/NodeIdentityConverterAspect.php#L34

At least we currently only anticipate it to be an `NodeAdress` object here https://github.com/neos/neos-development-collection/blob/a2dd063fde0c5b3ccf8c96b6b260870ad63f6d17/Neos.Neos/Classes/FrontendRouting/EventSourcedFrontendNodeRoutePartHandler.php#L265